code/modules/modular_computers/file_system/programs/security/penalmechs.dm

```diff
@@ -0,0 +1,113 @@
+/datum/computer_file/program/penal_mechs
+	filename = "penalmechs"
+	filedesc = "Penal Mech Monitoring"
+	program_icon_state = "security"
+	extended_desc = "This program allows monitoring and control of active penal miner mechs."
+	required_access_run = access_security
+	required_access_download = access_hos
```

Comment:
possible to make this warden level access? unless wardens get this 👀

Reply:
I've made it `access_armory` -- So far as I can tell that's the first unique access the warden gets that the lower level security peeps don't?